File: Java/LeetCode/Count Number of Pairs With Absolute Difference K.java
// https://leetcode.com/problems/count-number-of-pairs-with-absolute-difference-k/

class Solution {

    public int countKDifference(int[] nums, int k) {
        int pairCount = 0;
        
        for (int i = 0; i < nums.length; ++i) {
            for (int j = 0; j < nums.length; ++j) {
                if ( i < j && Math.abs(nums[i]-nums[j]) == k ) {
                    pairCount++;
                }
            }
        }
        
        return pairCount;
    }
    
}

File: Kotlin/LeetCode/Count Number of Pairs With Absolute Difference K.kt
// https://leetcode.com/problems/count-number-of-pairs-with-absolute-difference-k/

class Solution {

    fun countKDifference(nums: IntArray, k: Int): Int {
        var pairCount = 0;

        for (i in 0 until nums.size) {
            for (j in i+1 until nums.size) {
                if (abs(nums[i]-nums[j]) == k) {
                    pairCount++;
                }
            }
        }

        return pairCount;
    }

}
